packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -48,6 +48,16 @@ const DumbEditAccountModal = ({
    * vault modal), we delay a bit the appearing of the edit modal via CSS.
    */
   const shouldShow = useTimeout(500)
+
+  const fromReconnectButton = window.location.toString().endsWith('reconnect')
+  // If we come from the reconnect button so focus on secret field
+  const fieldsOptions = {
+    secret: {
+      forceEncryptedPlaceholder: !fromReconnectButton,
+      focus: fromReconnectButton
+    }
```

Comment:
I wonder if it is necessary to have the generic "fieldsOptions" name where we in fact handle only the secret field options. Here it seems that we can do 

```
const fieldsOptions = {
  login: {
    focus: true
  }
}
```

but it does not work. To me either we use a generic name and then make it so that there no surprises or we can use specific names:

```
const fieldOptions = { // Nit: I prefer in names to use only 1 plural "s"
  displaySecretPlaceholder: true,
  focusSecretField: true
}
```

Reply:
Agree with you